x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/index.tsx

```diff
@@ -6,108 +6,75 @@
-  canUserCreateAndReadCases,
-  formatBytes,
-  formatNumber,
-  ilmPhases,
-  lastChecked,
-  openCreateCaseFlyout,
-  patternIndexNames,
-  patternRollups,
-  patterns,
-  setLastChecked,
-  totalDocsCount,
-  totalIncompatible,
-  totalIndices,
-  totalIndicesChecked,
-  totalSizeInBytes,
-  onCheckCompleted,
-}) => {
-  const errorSummary = useMemo(() => getErrorSummaries(patternRollups), [patternRollups]);
+const DataQualitySummaryComponent: React.FC = () => {
+  const { isILMAvailable } = useDataQualityContext();
+  const { totalIndices, totalDocsCount, totalIndicesChecked, totalIncompatible, totalSizeInBytes } =
+    useResultsRollupContext();
 
   return (
-    <EuiPanel data-test-subj="dataQualitySummary" hasShadow={true}>
-      <EuiFlexGroup alignItems="flexStart" gutterSize="none" justifyContent="spaceBetween">
+    <EuiPanel paddingSize="l" data-test-subj="dataQualitySummary" hasShadow={true}>
+      <StyledFlexGroup
+        alignItems="stretch"
+        gutterSize="none"
+        justifyContent="spaceBetween"
+        wrap={true}
+      >
         <SummaryActionsContainerFlexItem grow={false}>
-          <SummaryActions
-            addSuccessToast={addSuccessToast}
-            canUserCreateAndReadCases={canUserCreateAndReadCases}
-            formatBytes={formatBytes}
-            formatNumber={formatNumber}
-            errorSummary={errorSummary}
-            ilmPhases={ilmPhases}
-            lastChecked={lastChecked}
-            onCheckCompleted={onCheckCompleted}
-            openCreateCaseFlyout={openCreateCaseFlyout}
-            patternIndexNames={patternIndexNames}
-            patterns={patterns}
-            patternRollups={patternRollups}
-            setLastChecked={setLastChecked}
-            sizeInBytes={totalSizeInBytes}
-            totalDocsCount={totalDocsCount}
-            totalIncompatible={totalIncompatible}
-            totalIndices={totalIndices}
-            totalIndicesChecked={totalIndicesChecked}
-          />
+          <SummaryActions />
         </SummaryActionsContainerFlexItem>
 
-        <EuiFlexItem grow={false}>
-          <StatsRollup
-            docsCount={totalDocsCount}
-            formatBytes={formatBytes}
-            formatNumber={formatNumber}
-            incompatible={totalIncompatible}
-            indices={totalIndices}
-            indicesChecked={totalIndicesChecked}
-            sizeInBytes={totalSizeInBytes}
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
+        <StyledFlexItem grow={false}>
+          {isILMAvailable && (
+            <StyledIlmPhaseFilterContainer>
+              <IlmPhaseFilter />
+            </StyledIlmPhaseFilterContainer>
+          )}
+          <StyledRollupContainer>
+            <StatsRollup
+              docsCount={totalDocsCount}
+              incompatible={totalIncompatible}
+              indices={totalIndices}
+              indicesChecked={totalIndicesChecked}
+              sizeInBytes={totalSizeInBytes}
```

Comment:
yes, because StatsRollup is a shared component and it receives differents sets of those props:

- Here it receives the total rollup stats
- In PatternSummary it receives pattern level rollup stats